src/depackers/inflate.c

```diff
@@ -1040,16 +1039,14 @@ int libxmp_inflate(FILE *in, FILE *out, uint32 *checksum, int is_zip)
 
   data.huffman_tree_len_static = NULL;
 
-  huffman_tree_len=malloc(HUFFMAN_TREE_SIZE * sizeof(struct huffman_tree_t));
+  huffman_tree_len=(struct huffman_tree_t *) malloc(HUFFMAN_TREE_SIZE * sizeof(struct huffman_tree_t));
   if (huffman_tree_len == NULL)
     goto err;
-
```

Comment:
See previous comment.

Reply:
There is no readability issue with this, it just groups allocation+memset together

Reply:
kept this one.